hbp_knowledge/er-stress/Karen2011.bel

```diff
@@ -147,29 +147,12 @@ p(HGNC:BAX) -> act(p(HGNC:ERN1), ma(kin))
 p(HGNC:BAK1) -> act(p(HGNC:ERN1), ma(kin))
 complex(p(HGNC:ERN1),p(HGNC:BAX),p(HGNC:BAK1))
 
-SET Evidence = "Protein tyrosine phosphatase
-1B (PTP1B), which is present mostly in the ER, also influences
-IRE1 activity. The absence of PTP1B caused impaired XBP1
-splicing, JNK phosphorylation and attenuated up-regulation of
-XBP1 target genes such as ER degradation enhancing alpha-mannosidase–
-like protein (EDEM) [12]."
-## Unable to find PTP1B in databases
-
 SET Evidence = "ASK1-interacting protein 1 (AIP1)
 was recently shown to specifically regulate and enhance IRE1
 signalling. [12]."
 
 complex(GO:"AIP1-IRE1 complex") -> act((p(HGNC:ERN1)))
 
-
-SET Evidence = "The IRE1 pathway
-is negatively modulated by the ER-located protein BAX
-inhibitor-1 (BI-1). BI-1 forms a protein complex with IRE1, and
-BI-1-/- cells show hyperactivation of IRE1 and a subsequent
-increase in XBP1 mRNA splicing [12]."
-
-## Unable to find BI-1 in databases
```

Comment:
[Google search](https://www.google.com/search?q=bax+inhibitor-1&oq=BAX+inhibitor-1&aqs=chrome.0.0l4.269j0j7&sourceid=chrome&ie=UTF-8) had answer in top 5 results in link to UniProt